src/lib/components/sidebar.svelte

```diff
@@ -4,26 +4,26 @@
 	import { onMount } from 'svelte';
 	let successCriteriaMap = {};
 	let criteriaPerPrincipe = {};
+	let baseUrl = `/${urlData.url.website.slug}/${urlData.url.slug}`;
+
+	console.log(urlData);
+	const criteriaSlice = urlData.url.checks.flatMap((check) =>
+		check.succescriteria.map((criteria) => criteria.index)
+	);
+
+	criteriaSlice.forEach((index) => {
+		const principleIndex = index.split('.')[0];
+		if (!successCriteriaMap[principleIndex]) {
+			successCriteriaMap[principleIndex] = [];
+		}
+		successCriteriaMap[principleIndex].push(index);
+	});
 
-	onMount(() => {
-		const criteriaSlice = urlData.url.checks.flatMap((check) =>
-			check.succescriteria.map((criteria) => criteria.index)
+	principes.forEach((principe) => {
+		criteriaPerPrincipe[principe.index] = principe.richtlijnen.reduce(
```

Comment:
Ik krijg echt jeuk van de combinatie Nederlands en Engels, niet jouw issue; maar zorg ajb dat alles gewoon in het Engels wordt opgeslagen in de API in jouw toekomstige projecten.

Reply:
Daar ga ik zeker op letten!